src/commands/login.ts

```diff
@@ -14,7 +14,7 @@ export const login = (username: string, password: string): Promise<void> => {
             logInfo(`Logged in as ${username}.`);
             exit(0);
         })
-        .catch(err => {
+        .catch((err) => {
```

Comment:
I'm surprised it didn't complain about needing a non-implicit any.

Reply:
Might be a newer-TypeScript thing than TSLint... it'll probably complain eventually ;)